03_gamedesign/index.tex

```diff
@@ -1 +1,139 @@
-\chapter{Game Design}
\ No newline at end of file
+\chapter{Game Design}
+
+\section{Implementation}
+\label{sec:GD:implementation}
+
+Implementation design was important as the majority of the class were involved in writing simulation code. As such, dedicated \emph{infrastructure} engineers from each team were elected to form an infrastructure team responsible for building the central part of the game, otherwise known as the \emph{server}. Each team's infrastructure engineer was tasked to own the implementation of a slice of the server.
+
+One such sub-team was the \textbf{core} infrastructure team, responsible for building the core parts of the game server. Inputs were taken from the entire class to choose the best implementation strategies and design--a solid and simple core foundation was paramount to allow clean continuous integration of code and ideas from all contributors. The subsections below detail implementation specifics of the game's simulation.
+
+\subsection{Architecture}
+\label{sec:GD:implementation:arch}
+
+The structure of the game was closely modelled after a \emph{client-server} model\footnote{\url{https://en.wikipedia.org/wiki/Client-server_model}}, but note ``closely''--whilst the nomenclature was taken directly from the aforementioned model, there are some differences. \emph{Server} and \emph{client} in this context bear the following definitions:

```

Comment:
is there potentially a more academic citation than w i k e p i d i a

Reply:
yes we shall put that in after #18

Reply:
#21 created for this